src/python/pants/core/goals/test.py

```diff
@@ -1029,8 +1055,8 @@ def _format_test_summary(result: TestResult, run_id: RunId, console: Console) ->
         elapsed_secs = total_elapsed_ms / 1000
         elapsed_print = f"in {elapsed_secs:.2f}s"
 
-    suffix = f" {elapsed_print}{source_desc}"
-    return f"{sigil} {result.description} {status}{suffix}."
+    suffix = f"{elapsed_print}{source_desc}"
+    return f"{sigil} {result.description} {status}{attempt_msg} {suffix}."
```

Comment:
Hm. I'm not sure why the `suffix` was broken out like this... might as well inline it below?

Reply:
huh, yeah. I guess because both parts of suffix could be empty, so it might have been intended to omit the space in case there was nothing after it? (although it doesn't do that). Anyhow, easy enough to inline.